common/info.yaml

```
@@ -297,6 +313,295 @@ description: |
+      < 201 OK {"_post_id": "0000000000000001"}
+
+      > POST http://TWCTBXX00003/v2/groups {"_post_id": "0000000000000001", "name": "Kitchen Lights", "devices": [1]}
+      < 201 OK {"_post_id": "0000000000000001"}
+
+  The last two requests could be parallelized.
+
+  Alternatively the client could pre-generate the ID and parallelize all three
+  requests. However, the client MUST be sure to use a good random number
+  generator and generate a 64-bit (16-nibble) hex string in lower-case
+  with leading zeros.
+
+  ### Group Deletion
+
+  To delete a group, the client should send a DELETE request to each
+  of the shards. If one of the Bonds is offline, then the group will only
+  be partially deleted. The user must bring that Bond back online in order
+  to finish deleting the group. In the meantime, the group will still exist
+  containing just the devices on the offline Bond.
+
+  ### Removing Device from Group
+
+  Devices may be removed from the Group via DELETE `groups/{}/devices/{}`.
+  When the last device is removed from a group, the shard containing that
+  device will disappear (an implicit deletion).
+
+  ### Changing Group Name
+
+  To change the name of a group, the client should send a PATCH request
+  to each of the shards. The name can be changed as long as there is at least
+  one online Bond participating in the group. If any Bonds are offline,
+  their `group/{}` endpoints will still have the old name. However, this
+  does not pose a problem for API clients because the `name` should be taken
+  from the group shard with the most recent `__modified` field.
+
+  The Bond Home app performs a special downward synchronization whereby
+  out-of-date group shard names are updated with the latest. That is,
+  when Bonds previously offline come back online, they are PATCHed with the
+  name of the most recent shard in the group. Other API clients do not
+  need to perform this downward sync function.
+
+  ## Skeds
+
+  Schedules on groups and scenes need to be installed into each shard,
+  and the Bond Home app performs downward synchronization on the skeds.
+
+  ### Sked Creation
+
+  Similar to group creation, Bond Home app creates the sked on each shard
+  **using the same 64-bit ID** for that sked on each shard. The sked ID
+  must be different than the group ID. Example:
+
+      > POST http://TWCTBXX00001/v2/groups/0000000000000001/skeds {"action":"SetBrightness", "argument":75, "mark":"sunrise", "seconds":0}
+      < 201 OK {"_post_id": "0000000000000007"}
+
+      > POST http://TWCTBXX00002/v2/groups/0000000000000001/skeds {"_post_id": "0000000000000007", "action":"SetBrightness", "argument":75, "mark":"sunrise", "seconds":0}
+      < 201 OK {"_post_id": "0000000000000007"}
+
+      > POST http://TWCTBXX00003/v2/groups/0000000000000001/skeds {"_post_id": "0000000000000007", "action":"SetBrightness", "argument":75, "mark":"sunrise", "seconds":0}
+      < 201 OK {"_post_id": "0000000000000007"}
+
```

May be good to add the parallelization info here too.
```suggestion

  The last two requests could be parallelized.

```